DataFormats/TotemRawData/interface/TotemRawEvent.h

```diff
@@ -0,0 +1,121 @@
+/****************************************************************************
+*
+* This is a part of the TOTEM offline software.
+* Authors: 
+*   Jan Kašpar (jan.kaspar@gmail.com) 
+*    
+****************************************************************************/
+
+#ifndef DataFormats_TotemRawData_TotemRawEvent
+#define DataFormats_TotemRawData_TotemRawEvent
+
+#include <ctime>
+#include <map>
+
+/**
+ * Encapsulates meta-data (non-VFAT data) of TOTEM raw event.
+**/
+class TotemRawEvent
+{
+  public:  
+    TotemRawEvent() {}
+
+    ~TotemRawEvent() {}
+
+    ///Additional data provided in OptoRx headers and footers.
+    struct OptoRxMetaData
+    {
+      unsigned int BX, LV1;
+    };
+
+    /// Trigger data provided by LoneG.
+    struct TriggerData
+    {
+      unsigned char type;
+      unsigned int event_num, bunch_num, src_id;
+      unsigned int orbit_num;
+      unsigned char revision_num;
+      unsigned int run_num, trigger_num, inhibited_triggers_num, input_status_bits;
+    };
+
+    unsigned long& getDataEventNumber()
```

Comment:
is there a good reason to expose data members through non-const references?
setXYZ methods would be more appropriate.